src/java.desktop/share/native/libsplashscreen/libpng/png.h

```diff
@@ -24,14 +24,9 @@
 
 /* png.h - header file for PNG reference library
  *
- * This file is available under and governed by the GNU General Public
- * License version 2 only, as published by the Free Software Foundation.
- * However, the following notice accompanied the original version of this
- * file and, per its terms, should not be removed:
```

Comment:
You have lost the above 4 lines from every source file you updated.

Reply:
Thanks for catching it Phil.
Updated.